### PR TITLE
Update README file with link to resetting secrets

### PR DESCRIPTION
**Pull request by felixtheflex**
### Context

Update README file with link to confluence page on resetting application secrets and password

### Changes proposed in this pull request

Update README file with link to confluence page on resetting application secrets and password

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,6 +104,10 @@ bar
 
 Refer to the [settings file](config/settings.yml) for all the settings required to run this app
 
+## Resetting Secrets and passwords
+
+Please visit the confluence page [here](https://dfedigital.atlassian.net/wiki/spaces/BaT/pages/2351824901/Resetting+Secrets+for+Register-Trainee-Teachers) for instructions
```
Comment (dankmitchell) [inline README.md:109]:
```suggestion
Please visit the [confluence page](https://dfedigital.atlassian.net/wiki/spaces/BaT/pages/2351824901/Resetting+Secrets+for+Register-Trainee-Teachers) for instructions
```

**Review by dankmitchell — APPROVED** [review]
Very minor request. Not a blocker.

File: README.md (hunk 1 above)
Comment (felixtheflex) [reply README.md]: done...

**Review by dankmitchell — APPROVED** [review]
🚢

**Comment by felixtheflex** [comment]
[Add Read Me Documentation](https://trello.com/c/8eeUacIX/727-add-read-me-documentation)

**Pull request merged**